sql-cli/src/opensearch_sql_cli/__init__.py

```diff
@@ -22,4 +22,4 @@
 express or implied. See the License for the specific language governing
 permissions and limitations under the License.
 """
-__version__ = "1.13.0.0"
+__version__ = "1.0.0"
```

Comment:
i think we are still using four digits? 1.0.0.0

Reply:
thx, fixed